packages/components/pie-modal/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { LitElement, html } from 'lit'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Are these linting rules necessary? The dependencies don't look extraneous to me

Reply:
To avoid duplication we use the `lit` package (and others) from the monorepo, not one from the `pie-modal` package. That's why ESLint complains.

Reply:
There's an issue related to it: https://github.com/import-js/eslint-plugin-import/issues/1174

We should probably either disable that rule or fix it, instead of disabling eslint on a per line basis.